zipline/pipeline/mixins.py

```diff
@@ -20,6 +20,7 @@
 from zipline.utils.input_validation import expect_types
 from zipline.utils.sharedoc import (
     format_docstring,
+    PIPELINE_ALIAS_DOC,
```

Comment:
Can we call this `PIPELINE_ALIAS_NAME_DOC`?  I read this name as suggesting that it holds the entire docstring for Alias, which makes it confusing when it's templated in for just the `name` parameter.